cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/DoTransformer.java

```diff
@@ -47,8 +46,7 @@
                 result.add(argument);
             }
         } else if (rawArguments instanceof String) {
-            // one liner syntax
-            result = transformOneLinerArguments((String) rawArguments);
+            throw new RuntimeException("Task arguments should be defined using a standard YAML list.");
```

Comment:
throw an error when the value is not a list and not null - how about this:
```
        } else if (rawArguments != null) {
            throw new RuntimeException("Task arguments should be defined using a standard YAML list.");
        }
```

so we will allow cases:
```
    - CheckWeather:
        do:
          ops.test_op:
```
and
```
    - CheckWeather:
        do:
          ops.test_op: [....]
```